src/layout/section/columnedContentSection/index.tsx

```diff
@@ -8,6 +8,11 @@ const StyledColumnedContentSection = styled(ColumnedContentSection)`
     margin: ${space.xl} 0;
   }
 
+  & .kirk-columned-content-section-title {
```

Comment:
should not be there :)

Reply:
Not part of this PR (created via the rebase) ;)